### ISSUE TITLE
Allow local IERS file redux

### ISSUE THREAD

**Issue by jchiang87** [LSSTDESC/imSim#426]
For running at Theta, we need to reimplement the fix described at #200

**Comment by cwwalter** [comment]
It looks like relative to when we dealt with this before there is now a new option:

iers.conf.auto_max_age = None  
https://docs.astropy.org/en/stable/utils/iers.html

which stops the check but would require us to prestige it in the "Standard" area, so I think the previous 

iers.conf.iers_auto_url = 'file:/srv/finals2000A.all'

is still better assuming that it still works.

**Comment by jchiang87** [comment]
For this morning's tests at Theta, I did try adding just that line, but it still tried to download the file, so I needed to set the iers_auto_url to point to a local copy.

**Comment by jchiang87** [comment]
Apparently astropy also wants to download a leap second file:
```
Downloading https://hpiers.obspm.fr/iers/bul/bulc/Leap_Second.dat
|===========================================| 1.3k/1.3k (100.00%)         0s
```
We can provide an alternative location by also setting `iers.conf.iers_leap_second_auto_url` to a local file URL, but it's easier just to disable downloads entirely.  This can be done with
```
from astropy.utils import iers
iers.conf.auto_download = False
iers.conf.iers_degraded_accuracy = 'warn'
```

**Comment by jchiang87** [comment]
This effectively closed by #427
